lib/Doctrine/DBAL/Schema/Exception/IndexNameInvalid.php

```diff
@@ -12,7 +12,7 @@ final class IndexNameInvalid extends SchemaException
     public static function new(string $indexName) : self
     {
         return new self(
-            sprintf('Invalid index-name %s given, has to be [a-zA-Z0-9_]', $indexName),
+            sprintf('Invalid index-name "%s" given, has to be [a-zA-Z0-9_]', $indexName),
```

Comment:
Why is it index-name with a dash?

Reply:
No good reason as far as I can tell. I will change to `index name`.